querier_test.go

```diff
@@ -1116,6 +1155,24 @@ func TestChunkSeriesIterator_DoubleSeek(t *testing.T) {
 	testutil.Equals(t, float64(2), v)
 }
 
+// Regression for: https://github.com/prometheus/tsdb/pull/327
```

Comment:
maybe point this to https://github.com/prometheus/tsdb/issues/328

To be honest I think the test itself explains the desired behaviour so don't think we even need to link it to the issue.

Reply:
Removed.